### ISSUE TITLE
Nightly failure in ipa-replica-install - 389-ds-base-1.4.4.13-2.fc33

### ISSUE THREAD

**Issue by flo-renaud** [389ds/389-ds-base#4757]
A nightly test failed in freeipa, in PR #[884](https://github.com/freeipa-pr-ci2/freeipa/pull/884), see [log](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/49401252-ae1c-11eb-bcb6-fa163e472ef6) and [report](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/49401252-ae1c-11eb-bcb6-fa163e472ef6/report.html):
```
cls = <class 'ipatests.test_integration.test_dnssec.TestInstallDNSSECFirst'>
mh = <pytest_multihost.plugin.MultihostFixture object at 0x7f95a006a160>

    @classmethod
    def install(cls, mh):
        tasks.install_master(cls.master, setup_dns=False)
        args = [
            "ipa-dns-install",
            "--dnssec-master",
            "--forwarder", cls.master.config.dns_forwarder,
            "-U",
        ]
        cls.master.run_command(args)
        # Enable dns service on master as it has been installed without dns
        # support before
        Firewall(cls.master).enable_services(["dns"])
    
>       tasks.install_replica(cls.master, cls.replicas[0], setup_dns=True,
                              nameservers=None)
```
The test failed during `ipa-replica-install`, in a step restarting httpd service. From [ipareplica-install.log](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/49401252-ae1c-11eb-bcb6-fa163e472ef6/test_integration-test_dnssec.py-TestInstallDNSSECFirst-install/replica1.ipa.test/var/log/ipareplica-install.log.gz):
```
2021-05-06T04:13:51Z DEBUG Starting external process
2021-05-06T04:13:51Z DEBUG args=['/bin/systemctl', 'restart', 'httpd.service']
2021-05-06T04:13:56Z DEBUG Process finished, return code=1
2021-05-06T04:13:56Z DEBUG stdout=
2021-05-06T04:13:56Z DEBUG stderr=Job for httpd.service failed because the control process exited with error code.
See "systemctl status httpd.service" and "journalctl -xe" for details.
```
The [journal](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/49401252-ae1c-11eb-bcb6-fa163e472ef6/test_integration-test_dnssec.py-TestInstallDNSSECFirst-install/replica1.ipa.test/journal.gz) shows that ipa-http-kdcproxy crashed with a time limit contacting 389ds:
```
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: Traceback (most recent call last):
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:   File "/usr/libexec/ipa/ipa-httpd-kdcproxy", line 100, in _find_entry
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:     entries = self.con.get_entries(
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:   File "/usr/lib/python3.9/site-packages/ipapython/ipaldap.py", line 1456, in get_entries
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:     self.handle_truncated_result(truncated)
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:   File "/usr/lib/python3.9/site-packages/ipapython/ipaldap.py", line 1168, in handle_truncated_result
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:     raise errors.TimeLimitExceeded()
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: ipalib.errors.TimeLimitExceeded: Configured time limit exceeded
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: ipa-httpd-kdcproxy: ERROR    Unknown error while retrieving setting from ldapi://%2Frun%2Fslapd-IPA-TEST.socket: Configured time limit exceeded
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: Traceback (most recent call last):
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:   File "/usr/libexec/ipa/ipa-httpd-kdcproxy", line 100, in _find_entry
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:     entries = self.con.get_entries(
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:   File "/usr/lib/python3.9/site-packages/ipapython/ipaldap.py", line 1456, in get_entries
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:     self.handle_truncated_result(truncated)
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:   File "/usr/lib/python3.9/site-packages/ipapython/ipaldap.py", line 1168, in handle_truncated_result
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]:     raise errors.TimeLimitExceeded()
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: ipalib.errors.TimeLimitExceeded: Configured time limit exceeded
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: ipa: ERROR: Unknown error while retrieving setting from ldapi://%2Frun%2Fslapd-IPA-TEST.socket: Configured time limit exceeded
May 06 04:13:56 replica1.ipa.test ipa-httpd-kdcproxy[28394]: ipa-httpd-kdcproxy: ERROR    Unknown error while retrieving setting from ldapi://%2Frun%2Fslapd-IPA-TEST.socket: Configured time limit exceeded
```

The following errors are also in the journal:
```
May 06 04:13:56 replica1.ipa.test ns-slapd[28060]: [06/May/2021:04:13:56.507767049 +0000] - ERR - ldbm_back_add - Retry count exceeded in add
May 06 04:13:56 replica1.ipa.test ns-slapd[28060]: [06/May/2021:04:13:56.509520256 +0000] - DEBUG - replication - multimaster_mmr_postop - error 0 for operation 560.
May 06 04:13:56 replica1.ipa.test ns-slapd[28060]: [06/May/2021:04:13:56.510570248 +0000] - DEBUG - NSMMReplicationPlugin - process postop -  csn=60936d02000200030000 canceling operation csn
May 06 04:13:56 replica1.ipa.test ns-slapd[28060]: [06/May/2021:04:13:56.511242565 +0000] - ERR - dna-plugin - dna_update_shared_config - Unable to update shared config entry: dnaHostname=replica1.ipa.test+dnaPortNum=389,cn=posix-ids,cn=dna,cn=ipa,cn=etc,dc=ipa,dc=test [error 51]
```
Details:
Test executed on fedora33, with 389-ds-base-1.4.4.13-2.fc33

**Comment by flo-renaud** [comment]
Note: this is low priority, issue was seen only once so far.

**Comment by mreynolds389** [comment]
Going to close this for now, if it reoccurs then reopen the ticket
